src/QueryLogger.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Muffin\Webservice;
+
+class QueryLogger extends \Cake\Database\Log\QueryLogger
```

Comment:
@markstory great to hear. I'm unsure however about what log level would be used by SQL queries (I expect debug) and what message should be used. And at that point how do you make nice tables from a simple string as message?
![image](https://cloud.githubusercontent.com/assets/1240253/10234964/90c79d4a-6898-11e5-94d6-8e0fc81331a8.png)